hr_dms_field/security/security.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <data>
+        <record id="group_hr_public_dms" model="res.groups">
+            <field name="name">DMS Is accessible publicly on employee profile</field>
+            <field name="category_id" ref="base.module_category_hidden" />
+        </record>
+    </data>
+    <data noupdate="1">
+        <record id="base.group_user" model="res.groups">
```

Comment:
Shouldn't it be HR managers only?

Reply:
This group allows to show your dms documents on the hr.employee.public view (only your documents, obviously)

This model is not visible for hr roles (replaced by hr.employee).

However, sometimes, it makes sense to disable this

Reply:
OK, right